src/plugins/data/server/server.api.md

```diff
@@ -609,7 +609,7 @@ export class Plugin implements Plugin_2<PluginSetup, PluginStart> {
     // (undocumented)
     setup(core: CoreSetup, { usageCollection }: DataPluginSetupDependencies): {
         fieldFormats: {
-            register: (customFieldFormat: import("../common").FieldFormatInstanceType) => number;
+            register: (customFieldFormat: import("../public").FieldFormatInstanceType) => number;
```

Comment:
I'm uncertain why this change appears since I'm unable to find a corresponding code change. Perhaps there's an indirect change.